### DISCUSSION TITLE
Query regarding Work Experience

### DISCUSSION THREAD

**Discussion by mohith01** [snigi-gupta/AttendingUniversityAtBuffalo#65] [General]
Hello Snigdha, I am Mohit and I am an incoming student to University at Buffalo for Fall 22. 

First of all, Thank you for creating this amazing guide for us.

I would like to know how much does work experience matter when applying to jobs in US as some people have more than 1 year of experience in India while some of us are joining fresh out of undergrad.

Thank you

**Comment by snigi-gupta** [comment]
@mohith01 
Have you checked out my Github's discussion section? I think you will find answers to these and many more questions. 
Work experience matters a little, but not that much. I had 2.5 yrs of experience and the only benefit I had was that I could apply for SDE 1 roles too. 
For students like you who do not have industry experience yet, you usually apply to NewGrad roles
That being said, most companies consider a graduate student as a fresher, since you start again in the industry. 
I wouldn't worry much about having experience, but rather focus on gaining experience by doing projects, internships, research work, and practicing competitive programming. 
My best friend works in the same team as me as an SDE 1 and he was a new grad like you. So you see, it does not matter much of having experience in the industry from your homeland country.

If you have more follow-up questions, please post them here, and I would be happy to answer them!
